src/win32/findfile.c

```diff
@@ -189,6 +226,12 @@ int git_win32__find_system_dirs(git_buf *out, const wchar_t *subdir)
 			&buf, HKEY_CURRENT_USER, REG_MSYSGIT_INSTALL_LOCAL, subdir) && buf.size)
 		git_buf_join(out, GIT_PATH_LIST_SEPARATOR, out->ptr, buf.ptr);
 
+#ifdef WIN64
+	if (!win32_find_git_in_registry(
+			&buf, HKEY_LOCAL_MACHINE, REG_MSYSGIT_INSTALL_LOCAL, subdir) && buf.size)
+		git_buf_join(out, GIT_PATH_LIST_SEPARATOR, out->ptr, buf.ptr);
```

Comment:
Don't these directories exist on x86, too? At least you check them in `win32_find_git_in_path` unconditionally, too.

Reply:
There's a suble difference... See the definition of `REG_MSYSGIT_INSTALL`.

Reply:
Maybe I should make this a bit more clear...

Reply:
On x64 systems the following `win32_find_git_in_registry` call uses the `Wow6432Node` redirect (i.e., the x86 version of Git for Windows....